admin/data/sql/migrations/088.do.create-job-table.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TABLE [mtc_admin].[job] (
+  id        INT IDENTITY (1, 1) NOT NULL,
+  urlSlug [uniqueidentifier] NOT NULL DEFAULT (newid()),
+  input      [nvarchar](MAX) NOT NULL,
+  createdAt DATETIMEOFFSET(3)   NOT NULL DEFAULT GETUTCDATE(),
+  updatedAt DATETIMEOFFSET(3)   NOT NULL DEFAULT GETUTCDATE(),
+  version rowversion,
+  jobStatusCode char(3) NOT NULL,
+  jobTypeCode char(3) NOT NULL,
+  [output]    [nvarchar](MAX),
+  [errorOutput]    [nvarchar](MAX),
+);
+
+ALTER TABLE [mtc_admin].[job] WITH CHECK ADD CONSTRAINT [FK_job_jobStatus_code] FOREIGN KEY([jobStatusCode])
+REFERENCES [mtc_admin].[jobStatus] ([code])
```

Comment:
Do we really want to point a FK to a non-primary key column?

Reply:
@GeorgeChatzigiannis you should link to the primary key of jobStatus table

Reply:
@GuyHarwood @jon-shipley updated as well